test/Microsoft.IdentityModel.Protocols.OpenIdConnect.Tests/ConfigurationManagerTests.cs

```diff
@@ -330,6 +331,43 @@ public void ResetLastKnownGoodLifetime()
             TestUtilities.AssertFailIfErrors(context);
         }
 
+        [Fact]
+        public void TestConfigurationComparer()
+        {
+            TestUtilities.WriteHeader($"{this}.TestConfigurationComparer", "TestConfigurationComparer", true);
+            var context = new CompareContext();
+
+            var config1 = new OpenIdConnectConfiguration() { TokenEndpoint = Default.Issuer + "oauth/token", Issuer = Default.Issuer };
```

Comment:
>TokenEndpoint = Default.Issuer + "oauth/token" [](http://example.com/codeflow?start=61&length=46)

here and below this will produce: "[http://Default.Issuer.comoauth/token](http://Default.Issuer.comoauth/token)"

add in the '/' for a more valid URI.